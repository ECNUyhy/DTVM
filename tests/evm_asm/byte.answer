0000000000000000000000000000000000000000000000000000000000000034