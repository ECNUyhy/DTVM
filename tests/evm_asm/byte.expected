status: success
error_code: 0
stack: []
memory: '0000000000000000000000000000000000000000000000000000000000000034'
storage: {}
transient_storage: {}
return: '0000000000000000000000000000000000000000000000000000000000000034'
events: []
